content/blog/tina-is-in-beta.md

```diff
@@ -61,4 +74,13 @@ Speed and performance have been something we have been actively working on. We i
 
 TinaCMS was built-in with small modular packages, this meant that we relied heavily on React context, and the dependency mismatches from over-modularizing our toolkit, which led to many bugs related to missing context.
 
-Our open-source team created @tinacms/toolkit which incorporates the essentials of Tina all in one place. This simplifies everything for you as a user and Tina as a product.
\ No newline at end of file
+Our open-source team created @tinacms/toolkit which incorporates the essentials of Tina all in one place. This simplifies everything for you as a user and Tina as a product.
+
+## Vercel Integration
+
+We wanted to reduce the friction to almost zero when testing TinaCMS, so we worked on adding Vercel integration. This means if you sign up for an account, you can one click and deploy in minutes and start playing around with TinaCMS and Tina Cloud. 
```

Comment:
For now the Vercel integration looks to be only available for when you choose to fork the starter from the dashboard?